.github/actions/spelling/allow.txt

```diff
@@ -1,122 +1,51 @@
-abstract
 alignas
 alignof
-and
-as
+Appium
+appium
```

Comment:
Changes like this are mostly to ensure that an update to `spelling.yml` will work w/o having the two differing versions of check-spelling disagree about what should be in `expect.txt`. Once the `spelling.yml` changes are applied, if there are items in here that you'd rather be in `expect.txt`, you can kick them out of here.